datafusion/core/src/physical_plan/aggregates/group_values/mod.rs

```diff
@@ -42,6 +43,19 @@ pub trait GroupValues: Send {
 
     /// Emits the group values
     fn emit(&mut self, emit_to: EmitTo) -> Result<Vec<ArrayRef>>;
+
+    /// Try to reserve the capacity that at least a single [`RecordBatch`] can be inserted. The
+    /// accumulator may reserve more space to speculatively avoid frequent re-allocations. After
+    /// calling try_reserve, capacity will be greater than or equal to self.len() + additional if
```

Comment:
"self.len() + additional" is referenced in this comment but I don't see `additional` anywhere 🤔

Reply:
Updated